vanilla-framework/vanilla-brochure-theme
# Bump VF dep to 1.3.x

## Done

- Pull in latest version of Vanilla Framework
- Fix up any regressions falling out of VF bump
- Fixed breaking travis build

## QA

- Pull down code
- Run `./run test` and verify there are no errors
- Run `./run serve --watch`
- Check through patterns at http://0.0.0.0:8102/vanilla-brochure-theme/ and make sure none are broken
- Cross ref with Percy and ensure all visual changes are expected
- Ensure travis build passes

## Demo

http://vanilla-brochure-theme-bump-vf.demo.haus/vanilla-brochure-theme/
